qt/DetailsDialog.cc

```diff
@@ -1310,9 +1310,23 @@ void DetailsDialog::onIdleModeChanged(int index)
 
 void DetailsDialog::onIdleLimitChanged()
 {
-    //: Spin box suffix, "Stop seeding if idle for: [ 5 minutes ]" (includes leading space after the number, if needed)
-    QString const units_suffix = tr(" minute(s)", nullptr, ui_.idleSpin->value());
+    //: Spin box format, "Stop seeding if idle for: [ 5 minutes ]"
+    QString const units_format = tr("%1 minute(s)", nullptr, ui_.idleSpin->value());
```

Comment:
Now that there's more near-duplicate code in these 3 places, please move the common part to a helper method in Utils.{h,cc}, so that it's callable as e.g. `Utils::updateSpinBoxPrefixAndSuffix(ui_.idleSpin, QT_TR_NOOP("%1 minute(s)"), "%1");`, maybe choosing a better for it if you can :)))

Reply:
I was kinda hoping to avoid duplication with the `tr()` call as well, that's why my suggestion included passing the untranslated format string and then translating it inside the helper, where spinbox value is also easily available.